UserAgent/Backend/Components/SPI/ADCDAC.cs

```diff
@@ -129,15 +123,31 @@ public void setDACVoltage2(double voltage)
             this.setDACVoltage(voltage, this.CHANNEL_2);
         }
 
-        public void readADCVoltage(int times, byte channel)
+        public double readADCVoltage1()
+        {
+            return _adConvert.ReadADCVoltage(this.CHANNEL_1);
+        }
+
+        public double readADCVoltage2()
         {
-            if (channel < 1 || channel > 2) return;
+            return _adConvert.ReadADCVoltage(this.CHANNEL_2);
+        }
 
+        public double updateCurrentADCVoltage2Average(int times)
```

Comment:
Naja, hinsichtlich der Lib, will man evtl genau den AVG haben. Da die Methode einen sprechenden Namen hat und sie kommentiert wird, sehe ich eigentlich keine Probleme. Ausser, du meinst, die Methode wird in der Lib nicht benötigt.